TradeItIosEmsApi/TradeItIosEmsApiLib.h

```diff
@@ -88,6 +88,11 @@
 #import "TradeItPriceInfo.h"
 #import "TradeItCancelOrderRequest.h"
 
+// Transactions
+#import "TradeItTransactionsHistoryRequest.h"
+#import "TradeItTransactionsHistoryResult.h"
+#import "TradeItTransactionHistory.h"
```

Comment:
I think this should just be called `TradeItTransaction`

Reply:
Have these been added to both the SDK and SDK-Carthage targets?

Reply:
> I think this should just be called TradeItTransaction

Ok, I will rename it

> Have these been added to both the SDK and SDK-Carthage targets?

Yes